ScilifelabDataCentre/dds_cli
# Update documentation

> **Before submitting the PR, please go through the sections below and fill in what you can. If there are any items that are irrelevant for the current PR, remove the row. If a relevant option is missing, please add it as an item and add a PR comment informing that the new option should be included into this template.**

> **All _relevant_ items should be ticked before the PR is merged**

# Description

- [x] **Summary of the changes and the related issue:** Update of documentation with examples. The branch is called new_python because I started bumping versions.

To test:
1. Run `pip install -r requirements-dev.txt`
2. Go into `docs` directory
3. Run `make html` (or `make clean html` to build from scratch)
4. Run `open _build/html/index.html`

The browser should now open the documentation.

## Type of change


# Checklist:

## General

- [x] [Changelog](../CHANGELOG.md): New row added. Not needed when PR includes _only_ tests.

## Repository / Releases

- [x] Rebase / update of branch done

## Checks

- [x] CodeQL passes
- [x] Formatting: Black & Prettier checks pass
- Tests
  - [x] The tests pass
